provider/datagen/src/transform/icuexport/normalizer/mod.rs

```diff
@@ -33,8 +33,8 @@ macro_rules! normalization_provider {
             }
         }
 
-        impl IterableDataProviderInternal<$marker> for crate::DatagenProvider {
-            fn supported_locales_impl(&self) -> Result<Vec<DataLocale>, DataError> {
+        impl IterableDataProvider<$marker> for crate::DatagenProvider {
+            fn supported_locales(&self) -> Result<Vec<DataLocale>, DataError> {
                 Ok(vec![DataLocale::default()])
             }
```

Comment:
Note: I left the trivial impls directly with IterableDataProvider in the previous PR but accidentally migrated this trivial impl.